src/Mvc/benchmarkapps/BasicApi/BasicApi.csproj

```diff
@@ -36,7 +36,7 @@
   <ItemGroup Condition="'$(BenchmarksTargetFramework)' != ''">
 
     <FrameworkReference Update="Microsoft.AspNetCore.App" RuntimeFrameworkVersion="$(MicrosoftAspNetCoreAppPackageVersion)" />
-    <FrameworkReference Update="Microsoft.NETCore.App" RuntimeFrameworkVersion="$(MicrosoftNETCoreAppPackageVersion)" />
+    <FrameworkReference Update="Microsoft.NETCore.App" RuntimeFrameworkVersion="$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)" />
```

Comment:
@sebastienros - I think benchmarks server might still be using MicrosoftNETCoreAppPackageVersion, so probably need to leave this and all benchmarks apps untouched.

Reply:
I am passing this property by myself when I run the benchmarks, so please leave it. I'll rename it at some point to prevent any ambiguity.